Engine/plugin/agsplugin.cpp

```diff
@@ -922,6 +931,19 @@ bool pl_use_builtin_plugin(EnginePlugin* apl)
 #endif // IOS_VERSION
 #endif // BUILTIN_PLUGINS
 
+    for(std::vector<InbuiltPluginDetails>::iterator it = _registered_builtin_plugins.begin(); it != _registered_builtin_plugins.end(); ++it) {
+        if (strncmp(apl->filename, it->filename, strlen(it->filename)) == 0) {
```

Comment:
I was copying the behaviour of pl_use_builtin_plugin in agsplugin.cpp.  I assumed it was so we could ignore filename extensions.